### ISSUE TITLE
mangohud defaults missing (with no indication of an issue in logs)

### ISSUE THREAD

**Issue by theofficialgman** [flightlessmango/MangoHud#1155]
**Describe the bug**
I am running the latest version of mangohud built from master branch. The mangohud output is empty, there is only a black seethrough square shown in the top left of any application (opengl or vulkan) run with mangohud.

**List relevant hardware/software information**
- Linux Distribution: PiOS Bookworm ARM64
- MangoHud version: master branch
- GPU: V3D 4.2 / 7.1.7

**To Reproduce**
1. Build and install mangouhud:
```
sudo apt install meson glslang-tools python3-mako cmake
```
```
git clone https://github.com/flightlessmango/MangoHud.git --depth=1
cd MangoHud
meson build --prefix /usr -Dappend_libdir_mangohud=false -Dwith_xnvctrl=disabled
ninja -C build
sudo ninja -C build install
```
2. run application with mangohud:
```
sudo apt install mesa-utils vulkan-tools
```
eg: `mangohud --dlsym glxgears` or `mangohud vkcube`
3. See that the application runs fine but there is no mangohud output at all

**Expected behavior**
It works as on all other systems I have tested

**Screenshots**
![2023-10-13_18-29](https://github.com/flightlessmango/MangoHud/assets/28281419/4bdb83fc-8efa-4aad-99e5-37e7e7c2c614)
![2023-10-13_18-29_1](https://github.com/flightlessmango/MangoHud/assets/28281419/750fb63e-3dd7-4193-9f2b-1b7b3db063e1)

**Comment by gort818** [comment]
Well I see the mangohud window, let's see your conf file

**Comment by theofficialgman** [comment]
> Well I see the mangohud window, let's see your conf file

there is no conf file it is simply using mangohud defaults.
installing mangohud does not add one (it adds an `MangoHud.conf.example` but this is not an active conf file).

**Comment by gort818** [comment]
Well you can you add a conf file?
here is a basic one
```
legacy_layout=0
gpu_stats=1
cpu_stats=1
fps=1
ram=1
vram=1
```
name it MangoHud.conf
save it to /home/$USER/config/MangoHud/

**Comment by theofficialgman** [comment]
adding a conf file does make it show up. However I have NO conf file on my x86_64 system that I built and installed on (same git commit and options used), yet it works without one. This is also the case on every other system I have used mangohud on... the defaults should load.

**Comment by theofficialgman** [comment]
also weird thing is manoghud is not reporting any `[info]` status in the logs...

**Comment by gort818** [comment]
You must be running an old version on your other machines.
If you want info you can use `MANGOHUD_LOG_LEVEL=info` you can also set it to `MANGOHUD_LOG_LEVEL=debug`

**Comment by theofficialgman** [comment]
> You must be running an old version on your other machines. If you want info you can use

ah silly me. I built on my x86_64 system but forgot to install. So yes I was running an older version there.

Why were defaults removed from mangohud? that seems counter intuitive. the least that could be done would be add some text that says "no config file found, please add one" to the overlay.

**Comment by flightlessmango** [comment]
Defaults haven't been removed. I'm not sure why it's not showing on your end

**Comment by flightlessmango** [comment]
This appears to be a bug, I can repro it on my end

**Comment by flightlessmango** [comment]
@gort818  The bug was introduced here 5f05f67e2274fc2546489abc1d93fcc27019f6ce
Not sure why it's happening

**Comment by gort818** [comment]
Ok will take a look and fix it

**Comment by gort818** [comment]
Fixed here 81941faa8ca5ad967215c8456fa057baf1e35b34
